rrfs-test/testinput_expr/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -239,12 +192,12 @@ variational:
       npz: 60
       layout: [8,10]
       io_layout: [1,1]
-      ntiles: 1
+      ntiles: 1 
       fieldsets:
       - fieldset: DataFix/fix/dynamics_lam_cmaq.yaml
     diagnostics:
       departures: ombg
-  - ninner: 50
+  - ninner: 5
```

Comment:
the same question,  5 is a too few for optimization using real data sets.

Reply:
In fact "gradient norm reduction: 1e-60" is too strict /ambitious. 
I would recommend to use values like 1e-20 even 1e-10.

Reply:
See my response from above. This PR is only updating the yamls to use more data for the MPAS-JEDI tests. If you want to change the fv3-jedi yamls then I can do that but those files were not created in this PR (only copied).

Reply:
@SamuelDegelia-NOAA   As I asked  previously, did you mean  those files will always be the same ( so, just multiple copied of the same file, or you started from copying the original file, but they are independent to each other?    if they are not supposed to be exactly multiple copies of the same file  , only this file need to be taken care of.

Reply:
The files in `testinput_expr` started as copies of the older yamls in previous version of `testinput`. But then I removed the test reference section from the copies I made in `testinput_expr` since they are no longer being used for ctests. 

But as I mentioned in another response, I should probably just remove the fv3jedi files from `testinput_expr` since they are the same as in `testinput` (minus the test reference section).

Reply:
@SamuelDegelia-NOAA  OK.  I will see if there would be another PR to change those setup mentioned above. 
If yamls in testinput_expr is mainly for an copy for possible use/comparison, they could be there as they are .